src/mindlessgen/prog/config.py

```diff
@@ -735,6 +737,66 @@ def __init__(self, config_file: str | Path | None = None):
         if config_file:
             self.load_from_toml(config_file)
 
+    def check_config(self, verbosity: int = 1) -> None:
+        """
+        Checks ConfigClass for any incompatibilities that are imaginable
+        """
+
+        # lower number of the available cores and the configured parallelism
+        num_cores = min(mp.cpu_count(), self.general.parallel)
+        if self.general.parallel > mp.cpu_count():
+            warnings.warn(
+                f"Number of cores requested ({self.general.parallel}) is greater "
+                + f"than the number of available cores ({mp.cpu_count()})."
+                + f"Using {num_cores} cores instead."
+            )
+
+        if num_cores > 1 and self.postprocess.debug:
+            # raise warning that debugging of postprocessing will disable parallelization
+            warnings.warn(
+                "Debug output might seem to be redundant due to the parallel processes "
+                + "with possibly similar errors in parallel mode. "
+                + "Don't be confused!"
+            )
+
+        if verbosity > 0:
+            if num_cores > 1:
+                # raise warning that parallelization will disable verbosity
+                warnings.warn(
+                    "Parallelization will disable verbosity during iterative search. "
+                    + "Set '--verbosity 0' or '-P 1' to avoid this warning, or simply ignore it."
+                )
+
+            # Check for f-block elements in forbidden elements
+            if self.generate.forbidden_elements:
+                f_block_elements = set(range(56, 70)) | set(range(88, 102))
+                if any(
+                    elem not in f_block_elements
+                    for elem in self.generate.forbidden_elements
+                ):
```

Comment:
Is this the correct condition? What you want to test is if _**all**_ entries of `f_block_elements` are within `forbidden_elements`. If this is not the case, you print this warning. It seems to me that your condition is doing something else.

Summary ChatGPT from current code:
> This checks if any element in the self.generate.forbidden_elements list or set is not within the f_block_elements. If any of the forbidden elements are outside of the f-block, the condition is true.

Doesn't seem like this is what we want to have.